src/ast/eval.rs

```diff
@@ -88,7 +88,7 @@ impl Evaluate for Node {
     fn eval_with(&self, scope: &Scope) -> EvaluationResult {
         match self.token {
             Token::Operator(ref operator) => {
-                dbg!(&self.args);
+                //dbg!(&self.args);
```

Comment:
this could simply be removed, like the other `dbg!`

Reply:
Fixed in my latest commit